openapi/components/schemas/ExternalServiceSettings.yaml

```
@@ -3,9 +3,130 @@ description: Settings for external service integration.
-      syncJournalEntries:
-        type: boolean
-        description: Specifies if journal entries must be synchronized.
-        default: false
+      customer:
+        type: object
+        title: QuickBooksOnlineCustomerExternalServiceSettings
+        description: Customer settings.
+        properties:
+          sync:
+            type: string
+            description: Specifies how to synchronize customers between your Rebilly and QuickBooks Online account.
+            default: manually
+            enum:
+              - manually
+              - when-used
+              - always
+            x-enumDescriptions:
+              manually: Customer data is only synchronized when it is manually executed.
+              when-used: Customer data is synchronized when related entities, such as transactions or invoices are created or updated.
+              always: Customer data is synchronized automatically as data is created or updated.
+          displayName:
+            type: string
+            default: full-name
+            enum:
+              - id
+              - full-name
+              - organization-name
+      invoice:
+        type: object
+        title: QuickBooksOnlineInvoiceExternalServiceSettings
+        description: Invoice settings.
+        properties:
+          sync:
+            type: string
+            description: Specifies how to synchronize invoices between your Rebilly and QuickBooks Online account.
+            default: manually
+            enum:
+              - manually
+              - when-issued
+              - always
+            x-enumDescriptions:
+              manually: Invoice data is only synchronized when it is manually executed.
+              when-issued: Invoice data is synchronized when issued.
+              always: Invoice data is synchronized automatically as data it is issued or updated.
+          unearnedRevenueAccountId:
+            type: string
+            description: ID of the journal account where all products are created.
+            example: jrn_acc_01H32J1C97KWGRCKF1E1JY0QG8
+          taxesAccountId:
+            type: string
+            nullable: true
+            default: null
+            description: ID of the journal account where all taxes are created.
+            example: jrn_acc_01H32J1KGXSRDH0K8Q7YYA552T
+          department:
+            type: string
+            nullable: true
+            default: null
+            description: ID of the QuickBooks Online department that invoices are related to.
+            example: jrn_acc_01H32J1KGXSRDH0K8Q7YYA552T
+          itemName:
+            type: string
+          itemDescription:
+            type: string
+          itemSku:
+            type: string
+          itemLineDescription:
+            type: string
+          taxName:
+            type: string
+          taxDescription:
+            type: string
+          taxSku:
+            type: string
+          taxLineDescription:
+            type: string
```

suggestion: suggestions for missing property descriptions 
(I'm not 100% on some of these)
```suggestion
          itemName:
            type: string
            description: Name of the invoice item.
          itemDescription:
            type: string
            description: Description of the invoice item.
          itemSku:
            type: string
            description: Stock Keeping Unit (SKU) item.
          itemLineDescription:
            type: string
            description: Description of the invoice line item.
          taxName:
            type: string
            description: Name of the tax that is applied to the invoice.
          taxDescription:
            type: string
            description: Description of the tax that is applied to the invoice.
          taxSku:
            type: string
            description: Tax that is applied to the Stock Keeping Unit (SKU) item.
          taxLineDescription:
            type: string
            description: Description of the tax line item.
```

---

More like:

```suggestion
          itemName:
            type: string
            description: Name of the invoice item.
          itemDescription:
            type: string
            description: Description of the invoice item.
          itemSku:
            type: string
            description: Stock Keeping Unit (SKU) of the invoice item.
          itemLineDescription:
            type: string
            description: Description of the invoice line item.
          taxName:
            type: string
            description: Name of the tax that is applied to the invoice.
          taxDescription:
            type: string
            description: Description of the tax that is applied to the invoice.
          taxSku:
            type: string
            description: Stock Keeping Unit (SKU) of the tax that is applied to the invoice.
          taxLineDescription:
            type: string
            description: Description of the tax line.
```